### ISSUE TITLE
Cannot find kha2d.*

### ISSUE THREAD

**Issue by pshtif** [KTXSoftware/TPlayer#7]
Just installed the haxelib Kha version and tried to build this sample and it seems the kha2d classes are no longer part of the Kha codebase.

**Comment by RobDangerous** [comment]
Right, Kha2D is a separate project on haxelib and I forgot to put the library reference in.
When you install kha2d, update TPlayer and remove the empty Libraries/Kha2D folder, it should now work (will fix the problem with the Kha2D folder for the next release - Kha prefers local libraries and gets confused by the empty directory git creates).

**Comment by pshtif** [comment]
Thanks will try that.

I will go a little off topic here but I am evaluating Kha as a possible low level gate for multiplatform development, I don't actually need high level stuff not even sprite etc. just something that exposes GPU api multiplatformly and other stuff like interaction, sound, asset loading etc. as minimal as possible so we can build framework on top of it. Is Kha good candidate or is there a lot of unnecessary high level stuff.

**Comment by RobDangerous** [comment]
That's exactly why I put the kha2d stuff in another project. No unnecessary high level stuff here.
